src/Umbraco.Web.UI.Client/src/common/directives/umbeditor.directive.js

```diff
@@ -28,6 +28,10 @@ angular.module("umbraco.directives")
                 }
 
                 scope.propertyEditorView = umbPropEditorHelper.getViewPath(scope.model.view, scope.isPreValue);
+				
+				 if(scope.model.view == 'textbox'){

```

Comment:
This is ultra hacky, we need to come up with a better solution than this, i'll investigate soon.